UtopiaCity/wwwroot/js/CityAdministration/Edit.js

```diff
@@ -0,0 +1,140 @@
+﻿window.onload = function () {
+    var id = window.location.search?.substring(4);
```

Comment:
looks scary

Reply:
Maybe there are some better solutions to pass Id from one page to another )

Reply:
Could you please advice something here?